packages/backend/src/test/mockReports.ts

```diff
@@ -53,39 +103,35 @@ export const BALANCES: BalanceRecord[] = [
   },
 ]
 
-export const PROJECTS: ReportProject[] = [
-  {
-    projectId: ProjectId('arbitrum'),
-    type: 'layer2',
-    escrows: [
-      {
-        address: ARBITRUM_ESCROW_ONE,
-        sinceTimestamp: new UnixTime(0),
-        tokens: [fakeTokenInfo({ id: AssetId.DAI, decimals: 18 })],
-      },
-      {
-        address: ARBITRUM_ESCROW_TWO,
-        sinceTimestamp: new UnixTime(0),
-        tokens: [
-          fakeTokenInfo({ id: AssetId.DAI, decimals: 18 }),
-          fakeTokenInfo({ id: AssetId.ETH, decimals: 18 }),
-        ],
-      },
-    ],
-  },
+const FUTURE_BALANCES = BALANCES.map((balance) => ({
+  ...balance,
+  timestamp: NOW.add(1, 'hours'),
+}))
+
+const REPORTS = createReports(PRICES, BALANCES, PROJECTS)
+const FUTURE_REPORTS = createReports(FUTURE_PRICES, FUTURE_BALANCES, PROJECTS)
+// We need this mock for now, it ensures that the OP token is included in the
```

Comment:
You'll forget about this comment in a week, mark it `TODO` or something that sticks out